R/model_LaTeX.R

```diff
@@ -2,21 +2,17 @@ SIR_LaTeX <- function(mu) {
   switch(mu + 1, # See the function help for why this addition is used.
     helpText(
       r"(Susceptible $$\frac{dS}{dt} = - \beta \frac{ S I}{N^q}$$)",
-      r"(Infectious $$\frac{dI}{dt} = \frac{\beta S I}{N^q} - \gamma I $$)",
-      r"(Recovered $$\frac{dR}{dt} = \gamma I $$)",
+      r"(Infectious $$\frac{dI}{dt} = \frac{\beta S I}{N^q} - \gamma {I} $$)",
+      r"(Recovered $$\frac{dR}{dt} = \gamma {I} $$)",
       r"(Reproductive ratio $$R_0 =  \frac{\beta}{\gamma} S(0)^q$$)",
-      "q-Value", br(),
-      r"($$ 1, frequency-dependent $$)", br(),
-      r"($$ 0, density-dependent $$)"
+      "q-Value $$\\begin{equation} x = \\begin{cases} 1 & \\text{, } frequency − dependent \\\\ 0 & \\text{, } density − dependent \\end{cases}\\end{equation} $$"
     ),
     helpText(
       r"(Susceptible $$\frac{dS}{dt} =\mu_B N - \mu_D S - \beta \frac{ S I}{N^q}$$)",
-      r"(Infectious $$\frac{dI}{dt} = \frac{\beta S I}{N^q} - \gamma I - \mu_D I$$)",
-      r"(Recovered $$\frac{dR}{dt} = \gamma I - \mu_D R $$)",
+      r"(Infectious $$\frac{dI}{dt} = \frac{\beta S I}{N^q} - \gamma {I} - \mu_D I$$)",
+      r"(Recovered $$\frac{dR}{dt} = \gamma {I} - \mu_D R $$)",
       r"(Reproductive ratio $$R_0 =  \frac{\beta}{\gamma} S(0)^q$$)",
-      "q-Value", br(),
-      r"($$ 1, frequency-dependent $$)", br(),
-      r"($$ 0, density-dependent $$)"
+      "q-Value $$\\begin{equation} x = \\begin{cases} 1 & \\text{, } frequency − dependent \\\\ 0 & \\text{, } density − dependent \\end{cases}\\end{equation} $$"
```

Comment:
Please use raw strings when writing $\LaTeX{}$. Inside the raw string you will only have to write $\LaTeX{}$, and won't need to escape backslash characters which are used to call macros and commands.